CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+Thanks for contributing. You rock!
+
+# Submitting an Issue
+
+All issues are tracked as part of the [cesium-ion-community](https://github.com/AnalyticalGraphicsInc/cesium-ion-community) repository.
```

Comment:
There are two different issue URLs which one should people use?
- https://github.com/AnalyticalGraphicsInc/cesium-ion-community
- https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues

Reply:
Thanks, after talking it over, we decided against funneling everything to these repositories so I'm going to change this up some.